pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1601,48 +1601,32 @@ self: super: {
   # Too strict upper bounds on text
   lsql-csv = doJailbreak super.lsql-csv;
 
-  reflex-dom = lib.pipe super.reflex-dom [
-      (appendPatch
-        (fetchpatch {
-          name = "bump-reflex-dom-bounds.patch";
-          url = "https://github.com/reflex-frp/reflex-dom/commit/70ff88942f9d2bcd364e301c70df8702f452df38.patch";
-          sha256 = "sha256-xzk1+6CnfhEBfXdL5RUFbLRSn7knMwydmV8v2F2W5gE=";
-          relative = "reflex-dom";
-        })
-      )
-      (overrideCabal (drv: {
-        editedCabalFile = null;
-        revision = null;
-      }))
-    ];
+  inherit (
+    let
+      useUpstream = subdir: assert super.reflex-dom-core.version == "0.8.1.0" && super.reflex-dom.version == "0.6.3.1" ;
```

Comment:
I have a preference for using `lib.warnIf` and friends for this sort of thing since the asserts can make downstream overrides impossible. That being said, not a requirement currently, we have a lot of these currently and should do a “treewide” change porting them over at some point.

Reply:
Done